core/themes/bartik/css/style.css

```diff
@@ -1185,6 +1185,32 @@ ul.action-links li a {
   padding: 0;
 }
 
+ /* -------------- Hero blocks  -------------- */
+
+.hero-block {
+  position: relative;
+  padding: 8% 4%; /* All padding shrinks/grows according to available width. */
+  min-height: 128px;
```

Comment:
I don't think vh is a good idea because on smaller devices the hero might need to be 300% screen height to fit all the text that's on it.  I'll try the transition and see how that feels :)